### PR TITLE
Prep for 1.0.0-beta1 release

### PR DESCRIPTION
**Pull request by lwander**
https://github.com/kubernetes-client/java/issues/82

### CODE REVIEW

**Comment by lwander** [comment]
We had to manually bump the -SNAPSHOT version for each component because we're skipping 0.3.0

File: README.md [hunk 1]
```diff
@@ -125,10 +125,11 @@ All APIs and Models' documentation can be found at the [Generated client's READM
...
 
-|                   | Kubernetes 1.4 | Kubernetes 1.5 | Kubernetes 1.6 | Kubernetes 1.7 |
-|-------------------|----------------|----------------|----------------|----------------|
-| java-client 0.1.0 | +              | +              | ✓              | -              |
-| java-client 0.2.0 | +              | +              | ✓              | -              |
+|                         | Kubernetes 1.4 | Kubernetes 1.5 | Kubernetes 1.6 | Kubernetes 1.7 |
+|-------------------------|----------------|----------------|----------------|----------------|
+| java-client 0.1.0       | +              | +              | ✓              | -              |
+| java-client 0.2.0       | +              | +              | ✓              | -              |
+| java-client 1.0.0-beta1 | +              | +              | ✓              | -              |
```
Comment (mbohlool) [inline README.md]: Should we try to update client for 1.8 before the release. It should be straightforward, right?

Comment (lwander) [reply README.md]: Oh good idea - go for it!

Comment (lwander) [reply README.md]: I'll just rebase this over the updated client when ready

Comment (mbohlool) [reply README.md]: Sure, I can do it.

**Comment by lwander** [comment]
Rebased, ready to release?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+# 1.0.0-beta1
+
+This is the first beta release of this client.
```
Comment (mbohlool) [inline CHANGELOG.md]: it worth mentioning that the client is now supports kubernetes 1.8 clusters. otherwise looks good.

**Comment by mbohlool** [comment]
Just one comment otherwise good to go. ping me on an online channel when it is ready to merge.

File: CHANGELOG.md (hunk 2 above)
Comment (lwander) [reply CHANGELOG.md]: Oh good call

**Pull request merged**